resources/shared/params.mjs

```diff
@@ -1,3 +1,5 @@
+export const LAYOUT_MODES = Object.freeze(["getBoundingClientRect", "elementFromPoint"]);
```

Comment:
It's a bit confusing that "elementFromPoint" does both getBoundingClientRect and elementFromPoint.
Maybe that should be reflected in the enum name? e.g. onlyBoundingRect, boundingRectAndElementFromPoint